js/apps/system/_admin/aardvark/APP/frontend/js/views/queryView.js

```diff
@@ -2211,12 +2211,7 @@
                 }
               } else if (geometry.type === 'Polygon' || geometry.type === 'LineString' || geometry.type === 'MultiLineString' || geometry.type === 'MultiPolygon') {
                 try {
-                  geojson = new L.GeoJSON(geometry, {
```

Comment:
is L.GeoJSON inbuilt, or was it another leaflet package? If it was and it's unused now, maybe we can remove it?

Reply:
I guess that specific package can still remain, but we can move others (like leaflet itself) to npm? Should work, but will need to try